python/paddle/fluid/layers/control_flow.py

```diff
@@ -933,6 +933,7 @@ def while_loop(cond, body, loop_vars, is_test=False, name=None):
         body(Callable): A callable returning a tuple or list of tensors and LoDTensorArrays of the same arity
```

Comment:
tensors and LoDTensorArrays
tensors **or** LoDTensorArrays?

Reply:
Done.